proto/c1/config/config.proto

```
@@ -0,0 +1,90 @@
+message BoolField {
+    bool default_value = 1;
+    optional BoolRules rules = 2;
+}
+
+message StringSliceField {
+    repeated string default_value = 1;
+    optional RepeatedStringRules rules = 2;
+}
+
+enum StringFieldType {
+    STRING_FIELD_TYPE_TEXT = 0;
```

Buf (1.47.2)

Enum zero value name "STRING_FIELD_TYPE_TEXT" should be suffixed with "_UNSPECIFIED".

(ENUM_ZERO_VALUE_SUFFIX)